cmd/upgrade.go

```diff
@@ -159,11 +171,6 @@ func (uc *upgradeCmd) loadCluster(cmd *cobra.Command) error {
 		return errors.Wrap(err, "error getting list of available upgrades")
 	}
 
```

Comment:
This was adding the current version to the allowed upgrade matrix, making the Cmd validation pass. the upgrade process however was re validating the same thing, disallowing same-version upgrades, and skipping validation alltogether for HostedMaster. I chose to move the validation in the cmd and let the underlying library (operations/upgradecluster) do the upgrades as it pleases.